Source/Core/Common/NandPaths.cpp

```diff
@@ -54,6 +56,21 @@ void ShutdownWiiRoot()
 {
   if (!s_temp_wii_root.empty())
   {
+    u64 tmd_title_id = Movie::GetTitleId();
```

Comment:
And with the current implementation, I don't think this will work when launching Wii channels (WADs) rather than Wii discs, because ES_DIVerify is only called for Wii discs.